03week/mastermind.js

```diff
@@ -116,19 +117,18 @@ function validEntry(myGuess) {
   return myGuess.length === solution.length && myGuess.split('').every(myChar => letters.some(validLetter => myChar === validLetter));
 }
 
+// This i
 function mastermind(guess) {
   // solution = 'abca'; // Comment this out to generate a random solution
   guess = guess.toLowerCase().trim();  // this is a "cleanup" statement will change all letters to lowercase and remove spaces.
 
-  if (validEntry(guess)) {
+  if (validEntry(guess)) {  // Testing to make sure
+    board[board.length] = `${guess}  :  ${generateHint(guess)}`;
     if (guess === solution) {
-      board[board.length] = `${guess}  :  ${generateHint(guess)}`;
       return true;  // player won!
```

Comment:
If you want  to pass the tests your need to return "You guessed it!" on line 128 but I see you did something similar on line 146...